monkey/infection_monkey/utils.py

```diff
@@ -32,6 +36,26 @@ def is_windows_os():
     return sys.platform.startswith("win")
 
 
+def get_host_info():
+    return {'hostname': socket.gethostname(), 'ip': get_primary_ip()}
+
+
+def get_primary_ip():
```

Comment:
What does primary IP mean? Why do you need it?

Reply:
It's the IP that machine would use to connect to the network. I need it to grab a usable (not 127.0.0.1) IP that would represent that machine.

Reply:
Then your lookup is wrong. Please look at how we do this elsewhere.